app/smc/pytest/e2e_smoke.py

```diff
@@ -279,8 +279,8 @@ def arc_watchdog_test(asic_id):
         # does not currently have specific exception types
         logger.error("DMC reset too early, ARC should still be online")
         return False
-    # Delay 600 more ms. Make sure the ARC has been reset.
-    time.sleep(0.6)
+    # Delay 1000 more ms. Make sure the ARC has been reset.
```

Comment:
While this isn't technically invalid, it does suggest something might be up with the watchdog functionality? The 0.6 second delay plus 0.5 second stop earlier should result in the ARC being reset- we ask for a 1000 ms timeout expiry, I would expect the timeout to expire within 1100 ms. We may need to see if something changed with the DMC timing code.